### PR TITLE
Account Details Screen

### PR DESCRIPTION
**Pull request by borisyankov**
* Add pressable avatars
* Account Details for users on Avatar press

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@borisyankov, it looks like you've already signed the Dropbox CLA. Thanks!

File: src/message/MessageFull.js [hunk 1]
```diff
@@ -49,3 +55,10 @@ export default class MessageFull extends React.PureComponent {
     );
   }
 }
+
+export default connect(
```
Comment (neerajwahi) [inline src/message/MessageFull.js]: Is this connect necessary? Doesn't seem like the state or callbacks are being used -- or am I missing something?

Comment (borisyankov) [inline src/message/MessageFull.js]: Good catch.  I think we did use it before, but not anymore. Would be nice for an eslint rule to exist for this. Will remove.

Comment (borisyankov) [inline src/message/MessageFull.js]: Ah, we need the actions part, but not the state. Fixing.

**Pull request merged**